source/skins/simple/OptionsSkin.js

```diff
@@ -120,6 +121,8 @@ export const OptionsSkin = (props: Props) => {
   const optionsStyle = optionsMaxHeight == null ? null : {
     maxHeight: `${optionsMaxHeight}px`
   };
+  const OPTION_HEIGHT = 46; // a single option's height is 46px by default
```

Comment:
i guess this should be configurable if you want to use any custom options, right?

Reply:
Sure, let's make that a prop!